release/pipeline/dags/istio_common_dag.py

```diff
@@ -88,9 +88,11 @@ def getBashSettingsTemplate(extra_param_lst=[]):
   template_list = [template_prefix]
   for key in keys:
     template_list.append("export %s={{ settings.%s }}" % (key, key))
-  template_list.append("""cp /home/airflow/gcs/data/airflow_scripts.sh ./airflow_scripts.sh
-                          chmod u+x                                    ./airflow_scripts.sh
-                          source                                       ./airflow_scripts.sh""")
+  template_list.append("""
+                cp /home/airflow/gcs/data/airflow_scripts.sh         ./airflow_scripts.sh
+                cp /home/airflow/gcs/data/airflow_scripts.sh.$BRANCH ./airflow_scripts.sh
```

Comment:
should the files be something like /home/airflow/gcs/data/airflow_scripts_$BRANCH.sh?

Or better yet, can we not have branch specific scripts and control them via parameters coming from here instead?

Reply:
the goal is to not have specific scripts, but the difference is too big between old ways of building with green build and without. Once we stop supporting 1.0 we wont need the $BRANCH

Reply:
the difference is in whether we use green-build repo or not. and that makes THE difference